src/opencensus_influxdb.erl

```diff
@@ -28,7 +28,7 @@ build_tags(Tags) ->
     lists:join($,, List).
 
 normalise(Atom) when is_atom(Atom) ->
-    normalise(erlang:atom_to_binary(Atom, utf8), <<>>);
+    normalise(erlang:atom_to_list(Atom, utf8), <<>>);
```

Comment:
Why this change?

Reply:
same question, it seems all normalization is done on binaries anyway. Might as well keep the binary as it is

Reply:
TBH I do not remember. Probably because first version used list to normalise instead of binary.